src/components/Tabs.react.js

```diff
@@ -0,0 +1,258 @@
+      this.props.children = [this.props.children];
+    }
+
+    EnhancedTabs = this.props.children.map((child, index) => {
+      // TODO: handle components that are not dcc.Tab components (throw error)
+      // enhance Tab components coming from Dash (as dcc.Tab) with methods needed for handling logic
+      let childProps;
+
+      if (child.props.children) {
+        // if props appears on .children, props are coming from Dash
+        childProps = child.props.children.props;
+      } else {
+        // else props are coming from React (Demo.react.js)
+        childProps = child.props;
+      }
+      return (
+        <EnhancedTab
+          key={index}
+          index={index}
+          id={childProps.id}
+          label={childProps.label}
+          selected={this.state.selected === index}
+          selectHandler={this.selectHandler}
+          className={childProps.className}
+          style={childProps.style}
+          selectedClassName={childProps.selected_className}
+          selected_style={childProps.selected_style}
+          value={childProps.value}
+          mobile_breakpoint={this.props.mobile_breakpoint}
+        />
+      );
+    });
+
+    const tabContainerClass = this.props.vertical
+      ? 'tab-container tab-container--vert'
+      : 'tab-container';
+
+    const tabContentClass = this.props.vertical
+      ? 'tab-content tab-content--vert'
+      : 'tab-content';
+
+    const tabParentClass = this.props.vertical
+      ? 'tab-parent tab-parent--vert'
+      : 'tab-parent';
+
+    return (
+      <div
+        className={`${tabParentClass} ${this.props.parent_className || ''}`}
+        style={this.props.parent_style}
+      >
+        <div
+          className={`${tabContainerClass} ${this.props.className || ''}`}
+          style={this.props.style}
+        >
+          {EnhancedTabs}
+        </div>
+        <div
+          className={`${tabContentClass} ${this.props.content_className || ''}`}
+          style={this.props.content_style}
+        >
+          {this.props.children[this.state.selected].props.children}
+        </div>
+        <style jsx>{`
+          .tab-parent {
+            display: flex;
+            flex-direction: column;
+          }
+          .tab-container {
+          }
+          .tab-container--vert {
+            display: inline-flex;
+            flex-direction: column;
+          }
+          :global(.tab-container--vert .tab--selected) {
+            border: 1px solid ${greyColor};
+            border-left: 2px solid ${primaryColor};
+            border-right: none;
+          }
+          .tab-content {
+          }
+          .tab-content--vert {
+            display: inline-flex;
+            flex-direction: column;
+          }
+          @media screen and (min-width: ${this.props.mobile_breakpoint}px) {
+            .tab-parent--vert {
+              display: inline-flex;
+              flex-direction: row;
+            }
```

Comment:
This is very very cool. Huge props to leading the charge with styled components.

I think that stacking the tabs looks great.
![image](https://user-images.githubusercontent.com/1280389/42523151-e36f72a0-843a-11e8-8b83-2fe246586040.png)

Are you happy with how the mobile view turned out?